pkg/cmd/stack/deploy.go

```diff
@@ -555,6 +554,43 @@ func isAnyPortAvailable(ctx context.Context, svc *model.Service, stack *model.St
 	return false
 }
 
+func deployK8sService(ctx context.Context, svcName string, s *model.Stack, c kubernetes.Interface, spinner *utils.Spinner) error {
+	svcK8s := translateService(svcName, s)
+	old, err := services.Get(ctx, svcName, s.Namespace, c)
+	if err != nil {
+		if !oktetoErrors.IsNotFound(err) {
+			return fmt.Errorf("error getting ingress '%s': %w", svcName, err)
+		}
+		if err := services.Deploy(ctx, svcK8s, c); err != nil {
+			return err
+		}
+		spinner.Stop()
+		oktetoLog.Success("Endpoint '%s' created", svcName)
+		spinner.Start()
+		return nil
+	}
+
+	if old.GetLabels()[model.StackNameLabel] == "" {
+		oktetoLog.Warning("skipping deploy of kubernetes service %s due to name collision: the service '%s' was running before deploying your compose", svcName)
```

Comment:
I think you cannot use one single argument to convert 2 placeholders `%s`. You would need to do one of these:

`oktetoLog.Warning("skipping deploy of kubernetes service %s due to name collision: the service '%s' was running before deploying your compose", svcName, svcName)`

or 

`oktetoLog.Warning("skipping deploy of kubernetes service %[1]s due to name collision: the service '%[1]s' was running before deploying your compose", svcName)`